docs/docusaurus.config.js

```diff
@@ -17,6 +17,27 @@ along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 
 const lightCodeTheme = require('prism-react-renderer/themes/github');
 const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+const { join } = require('path');
+
+const packages = [
+	'web3',
+	'web3-eth',
+	'web3-eth-contract',
+	'web3-utils',
+	'web3-validator',
+	'web3-common',
+	'web3-core',
```

Comment:
What you thing should we include documenting web3.js internal packages as well.

Reply:
These are not just internal packages, these are public packages. So had to be included anyway.

Reply:
- These are not directly part of eth api
- also considering 1.x parity docs
- We will not test these packages as standalone for system tests

But we can keep them in doc for users ref, and can document on least priority as compared to eth api packages.

Reply:
We can't complete the Eth or any other documentation without documenting `web3-core` and `web3-common` packages. e.g. If we need to document property `web3.eth.defaultAccount` that is inherited from the `Web3Context` in `web3-core` package, so it must be documented. Similarly for other components in those shared packages.

Reply:
These properties are used and exposed via main API packages like eth , contract etc.